src/foam/nanos/http/LiveScriptBundler.java

```diff
@@ -0,0 +1,353 @@
+package foam.nanos.http;
+
```

Comment:
Please follow the FOAM style guide:
https://github.com/foam-framework/foam/wiki/CodingStyle

Reply:
Updated